ckanext/canada/validators.py

```diff
@@ -112,8 +112,9 @@ def user_read_only_json(key: FlattenKey,
 
     if hasattr(package, key[0]):
         data[key] = json.loads(getattr(package, key[0]))
-    else:
-        data[key] = json.loads(package.extras.get(key[0], 'None'))
+    elif hasattr(package, 'extras'):
+        # type_ignore_reason: checking attribute
+        data[key] = json.loads(package.extras.get(key[0], 'None'))  # type: ignore
```

Comment:
```suggestion
        data[key] = json.loads(package.extras.get(key[0], 'null'))  # type: ignore
```
None is invalid json

Reply:
Okay so this validator is only used in the `prop` (Suggested Datasets) dataset type. in the fields `Keywords` and `Subjects`.

This has been changed to `null` now.